### PR TITLE
Remove last page link from pagination on tag page

### PR DESCRIPTION
**Pull request by GHaberis**
## What does this change?

Remove last page link from pagination on tag pages.

This was an action that came out of an incident retro, Details can be found [here](https://docs.google.com/document/d/1OxXCRn0CUhBwuDO1obNkJ4M-MFD90CdwUfvd1nnriw4/edit?ts=5a70c012#).

The corresponding Trello card is [here](https://trello.com/c/K2hAExgU/30-remove-last-pagination-link-in-tag-pages).

## What is the value of this and can you measure success?

Should lighten load on CAPI.

## Does this affect other platforms - Amp, Apps, etc?

No

## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

No

## Screenshots

Before...

picture 131

After...

picture 130

## Tested in CODE?

Yes

### CODE REVIEW

**Comment by NataliaLKB** [comment]
Would we be able to keep the `...`? Right now it looks like there are only 4 pages in the screenshot when there are actually more.

**Comment by GHaberis** [comment]
@NataliaLKB interesting point, I did implement what you suggested but felt it looked imbalanced...

picture 132

I thought the next button alone might be enough, as it wouldn't be there if there were only 4 pages. Also if you click on page 4, you do the see the next page, for example:

picture 133

Maybe one for design/ux input?

**Review by SiAdcock — APPROVED** [review]
Assuming the UX is 👍 , the code looks 👍 👍

**Comment by GHaberis** [comment]
@SiAdcock I made the change suggested by @NataliaLKB . Here are the various states pagination could be in now...

picture 138

picture 139

picture 137

File: common/app/views/fragments/pagination.scala.html [hunk 1]
```diff
@@ -28,11 +28,7 @@
 
 @paginationExtreme(pageNo: Option[Int], dir: String) = {
```
Comment (SiAdcock) [inline common/app/views/fragments/pagination.scala.html]: Does this function still need to accept the second param? Looks like it's always called with `"first"`

Comment (GHaberis) [reply common/app/views/fragments/pagination.scala.html]: Yeah you're correct, will remove the 2nd param and the `@if(dir == "first")` check...

**Review by SiAdcock — APPROVED** [review]
Approved x2 👍 👍

**Comment by paperboyo** [comment]
Does the introduction of https://github.com/guardian/content-api/pull/2121 mean we can (one day?) revert(/reengineer) this?

**Comment by GHaberis** [comment]
@paperboyo unfortunately this doesn't resolve the issue, there's currently a bug with "next/prev branch" that the CAPI team are aware of, once this is resolved we can revert this. @regiskuckaertz will notify us once this is the case.

**Comment by paperboyo** [comment]
Good news! Thank you for the update, @GHaberis!

**Pull request merged**